cmake/modules/root.cmake

```diff
@@ -36,6 +36,14 @@ zephyr_file(APPLICATION_ROOT ARCH_ROOT)
 # Convert paths to absolute, relative from APPLICATION_SOURCE_DIR
 zephyr_file(APPLICATION_ROOT SCA_ROOT)
 
+if(SYSBUILD)
```

Comment:
I don't believe fetching of roots should be depending on whether sysbuild is in use.

Why should a regular Zephyr CMake build not ensure same order of precedence regarding roots, as is done when sysbuild is in use ?

Reply:
Fixed